mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerProvider.java

```diff
@@ -0,0 +1,99 @@
+/****************************************************************
```

Comment:
You commit say that you just remove all junit test and here you add this class why ?